_pages/ato/categorize.md

```diff
@@ -0,0 +1,32 @@
+---
+title: ATO Categorization
+navtitle: Categorize
+---
+
+### Overview
+
+This part of the process looks at the functionality and data of the system, and categorizes its impact on three metrics (`confidentiality`, `integrity`, and `availability`) with three impact levels (`low`, `moderate`, `high`).
+
+Determining the impact levels is subjective; it is either the designated Authorizing Official (AO) or full AO who owns the final decision.
+
+### Categorize
+
+To categorize your system, go through each of the metrics and determine your system's impact level. `low`, `moderate`, and `high` refer to the impact on either the government or the public if the metric is lost. The framework we usually use is to ask ourselves (and the agency we are creating the system with) three worst case scenario questions:
+
+* What is the worst possible outcome if all of the _confidentiality_ of the system is lost? i.e.
+    * What if all of the data/content in the system is exposed to the public?
+* What is the worst possible outcome if all of the _integrity_ of the system is lost? i.e.
+    * What if an error makes it into the data/content?
+    * What if an update to the data/content is lost?
+* What is the worst possible outcome if all of the _availability_ of the system is lost? i.e.
+    * What if the system has downtime?
+
+The answer to each question should then be interpreted in terms of impact to either the government or the public. The "high water mark" for either impacted party must be used.
```

Comment:
I mean, the AO has the final say on _everything_ (hence the "owns the final decision" disclaimer above), but the hope is that we can give enough guidance that they will come to the same conclusion.